config/locales/en.yml

```diff
@@ -71,7 +71,7 @@ en:
               <li><strong>a scan or photo</strong> of the original notice or review conclusion letter</li>
               <li><strong>reasons for your appeal</strong> to copy and paste or attach as a document</li>
             </ul>
-            <p>You will also need to complete an <a href="%{approval_form_href}" target="_blank" class="ga-pageLink" data-ga-category="Proforma" data-ga-label="rep auth">authorisation form (%{approval_form_format_and_size})</a> if you want someone to represent you and they are not a legal professional.</p>
+            <p>You will also need to complete an <a href="%{approval_form_href}" target="_blank" class="ga-pageLink" data-ga-category="Proforma" data-ga-label="rep auth" target="_blank">authorisation form (%{approval_form_format_and_size})</a> if you want someone to represent you and they are not a legal professional.</p>
```

Comment:
There seem to be 2 `target="_blank"` in here...or is that deliberate for some html tag magic?